script/Ciel/Day 13/13_01_02_CIEL13_8B.txt

```diff
@@ -156,17 +183,22 @@ It's not like we can be together all the time.
 [sha:71380e5f47284fe3b1db1322ccc4d9137ac711a3]{
 -- Page 518, Offset 36728.
 -- 今までずっとそうだったし、場合によってはこれからずっと、一緒にはいられない。
-That's how it's always been, and depending on how things go, we might not be able to stay together in the future.
+// 今までずっと / これからずっと kinda hard to rhyme in EN.
+// Apart/part here kind of a slant rhyme, but something like 'forever' is hard to work into a past context
+We've already spent most of our lives apart,
+ and depending on how things go,
+ we may be forced to part once again.
```

Comment:
Well, I'm not sure if it's critical, but kind of a fun sentence to fiddle around with

Here's an attempt.
>That's how it has always been, and depending on circumstance, maybe also how it will always be.

Reply:
Hmm, it kinda makes it sound like he's saying "It's always been the case that we can't be together _all of the time_", which is like, something that's trivially still true? I feel like the sentence should emphasize more that they won't be able to be together again _at all_

Reply:
Yeah, I agree it's not ideal. Feels like you'd want to at least make an attempt, but idk.